src/test/java/api/support/fixtures/LostItemFeePoliciesFixture.java

```diff
@@ -33,14 +32,14 @@ public IndividualResource facultyStandard() {
     patronBilledAfterAgedLost.put("intervalId", "Months");
 
     JsonObject chargeAmountItem = new JsonObject();
-    chargeAmountItem.put("chargeType", "Actual cost");
+    chargeAmountItem.put("chargeType", "anotherCost");
```

Comment:
Under what circumstances would there be a charge type of `anotherCost`?

Reply:
When user chose `Set cost` on UI when configuring lost item policy.

Reply:
I think this likely needs raising with Holly. I'll add it to the list of things I need to ask.

Reply:
I think this quirk of the design for the policy should be as hidden as possible from circulation, both in the test code and the production code.

Could the fixture use the builder for this policy?

Reply:
Is this also to be transferred over into a subsequent pull request?

Reply:
Yes, it is related changes.